keywords.robot

```diff
@@ -2978,3 +2978,26 @@ Should Contain All
     FOR    ${substring}    IN    @{substrings}
         Should Contain    ${string}    ${substring}
     END
+
+Flash SD Wire
+    [Documentation]    This keyword flashes the sd wire that's connected to
+    ...    the provided RTE IP.
+    Open Connection And Log In
+
+    Variable Should Exist    ${file_bmap}
+    Variable Should Exist    ${file_gz}
+
+    SSHLibrary.Put File    ${file_bmap}    /data/    scp=ON
+    SSHLibrary.Put File    ${file_gz}    /data/    scp=ON
+
+    # sonoff turn off power should be here
+
+    SSHLibrary.Execute Command    sd-mux-ctrl -e=sd-wire_01-80 --ts
+    SSHLibrary.Execute Command    umount /dev/sda*
+    SSHLibrary.Execute Command    bmaptool copy --bmap /data/${file_bmap} /data/${file_gz} /dev/sda
```

Comment:
@WiktorG351 each command should have separate pass/fail check, without this approach it will hard to debug the possible problems with procedure

Reply:
@artur-rs i added a [Get Status Of SD Wire](https://github.com/Dasharo/open-source-firmware-validation/blob/a5528a512d760cae06aa057a36a7ecb3293e5146/keywords.robot#L3034) that checks whether the command executes successfully. I am calling it each time after the test changes the status, precisely [here](https://github.com/Dasharo/open-source-firmware-validation/blob/a5528a512d760cae06aa057a36a7ecb3293e5146/keywords.robot#L2997-L2999) and [here](https://github.com/Dasharo/open-source-firmware-validation/blob/a5528a512d760cae06aa057a36a7ecb3293e5146/keywords.robot#L3004-L3006)